### ISSUE TITLE
Deploy command fails when use testnet

### ISSUE THREAD

**Issue by krivochenko** [ton-community/blueprint#36]
Hi. Thanks for great work of team of this project! But there is a issue, when I deploy to testnet.

OS: MacOS
Node: v16.15.1

1. `yarn blueprint run`
2. `Choose file to use deployCollection`
3. `Which network do you want to use? testnet`
4. `Which wallet are you using? TON Connect compatible mobile wallet (example: Tonkeeper)`
5. `Choose your wallet Tonkeeper`

I see QR, and scan it (of course before I have switched Tonkeeper to Testnet). After that I see error in console:

```
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '/<my project root>/temp/testnet/tonconnect/ton-connect-storage_http-bridge-gateway::https:/bridge.tonapi.io/bridge'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/<my project root>/temp/testnet/tonconnect/ton-connect-storage_http-bridge-gateway::https:/bridge.tonapi.io/bridge'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
As I see, `/<my project root>/temp/testnet/tonconnect` directory is always empty.

Thanks for your help.

**Comment by arterialist** [comment]
Try upgrading to blueprint 0.6.0

**Comment by krivochenko** [comment]
Now even when I create new project I get error:
```
Error: Cannot find module 'tvm-disassembler/dist/disassembler'
Require stack:
- /<my project root>/node_modules/@tact-lang/compiler/dist/pipeline/build.js
- /<my project root>/node_modules/@tact-lang/compiler/dist/main.js
- /<my project root>/node_modules/@ton-community/blueprint/dist/compile/compile.js
- /<my project root>/node_modules/@ton-community/blueprint/dist/cli/build.js
- /<my project root>/node_modules/@ton-community/blueprint/dist/cli/create.js
- /<my project root>/node_modules/@ton-community/blueprint/dist/cli/cli.js
```

**Comment by krigga** [comment]
There's currently an issue with tact which is included in blueprint 0.6.0

Here is a temporary fix:
1. Add this to package.json
```
"overrides": {
    "tvm-disassembler": "2.0.0"
}
```
2. Run `npm install`
3. Create the initial contract yourself uaing `npx blueprint create`

**Comment by krigga** [comment]
The issue in a dependency was fixed, you can now either run `npm install` in an existing project (without the `overrides` section in package.json), or create a new project using `npm create ton@latest`.

Please reopen this issue if the bug persists.
